code/modules/mob/living/carbon/superior_animal/robots/subtype/church/types/bishop.dm

```diff
@@ -22,3 +22,4 @@
 	drop1 = /obj/item/book/ritual/cruciform/priest
 	drop2 = /obj/item/stack/material/gold/random
 	armor = list(melee = 15, bullet = 15, energy = 10, bomb = 5, bio = 100, rad = 100, agony = 100)
+	range_telegraph = "starts to hum well staring at"
```

Comment:
```suggestion
	range_telegraph = "starts to hum ominously while staring at"
```
Ominously here because we should communicate the general feeling of what is about to happen